pkg/detectors/flyio/flyio.go

```diff
@@ -0,0 +1,104 @@
+package flyio
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"flyio"}) + `\b([a-zA-Z0-9_]{47})\b`)
```

Comment:
@lonmarsDev all the tokens I generated started with `fo1_`. Does that match what you were seeing? If so we should probably add `fo1_` as part of this regex and to the keywords.

Reply:
Fly.io has a few legacy token formats that are no longer issued, but still might exist in the wild:

- `[a-zA-Z0-9_-]{43}`
- `fm1[ar]_[a-zA-Z0-9+\/]={0,2}`

And two formats that we currently issue

- `fo1_[a-zA-Z0-9_-]{43}`
- `fm2_[a-zA-Z0-9+\/]={0,2}`